Tests/XCUITests/TabsPerformanceTests.swift

```diff
@@ -5,20 +5,26 @@
 import XCTest
 
 class TabsPerformanceTest: BaseTestCase {
-    let fixtures = ["testPerfTabs_1_20startup": "tabsState20.archive", "testPerfTabs_3_20tabTray": "tabsState20.archive", "testPerfTabs_2_1280startup": "tabsState1280.archive", "testPerfTabs_4_1280tabTray": "tabsState1280.archive"]
+    let fixtures = ["testPerfTabs_1_20startup": "tabsState20.archive",
```

Comment:
We should rename this test suite since it has more tests now not only those related to tabs

Reply:
done